docs/solana/migration/hotspot-maker.mdx

```diff
@@ -317,10 +317,69 @@ like to use the library, the setup guide can be found here
 
 ---
 
-## Data Credits and Sol for Onboarding
-
-<!-- TODO: We need to create an easy way for makers to get sol and data credits into their accounts -->
-
-```
-
-```
+## Data Credits and SOL for Onboarding
+
+:::info
+
+Maker Wallets will need both `SOL` and `DC` for transactions post Migration.
+
+:::
+
+### Maker Wallet SOL
+
+Maker Wallets will need `SOL` to pay transaction fees and for minting new merkle trees if creating
+double the current amount of Hotspots in their existing fleet.
```

Comment:
This is kinda confusing. What is meant by 'creating double the current amount of Hotspots'?

This is also the first time we mention a merkle tree which is going to be a foreign concept to most of these radio manufacturers.

Reply:
Maybe we just frame this as Hotspot manufacturers being responsible for 'minting' the Hotspot NFT? Should specify the rough per-Hotspot cost for this action.

Reply:
think i trimmed it down well, let me know how this looks